base/initializers/inn.ts

```diff
@@ -0,0 +1,51 @@
+import {initialize_interactable_objs_data} from "./interactable_objects";
+
+export class Inn {
```

Comment:
Following the pattern that we currently have, would be nice if this `Inn` class got moved to a separate file (`Inn.ts`), just like `Shop.ts`.

Reply:
Ok